src/test/java/org/ccci/util/mail/EmailAddressTest.java

```diff
@@ -16,4 +26,65 @@ public void testValidation_noTrailingSpaces()
     {
         EmailAddress.valueOf("matt.drees@ccci.org ");
     }
+
+    @Test

```

Comment:
Tests for emails to ensure they are working as expected, test will be removed before deploy.

Reply:
Why are you wanting to remove the tests?

Reply:
Both security and it requires an email be sent then read to verify that it works as expected.

Reply:
Will these tests be run in GHA and fail without the necessary secrets?
If that is the case it would be preferable to configure/annotate them (somehow) to only run when explicitly invoked, so that they can be left in the code base but hinder GHA deploy.

I do something like that for integration tests in ERT
https://github.com/CruGlobal/ert-api/blob/master/eventhub-api-webapp/pom.xml#L574
https://github.com/CruGlobal/ert-api/blob/master/eventhub-api-webapp/src/test/kotlin/org/cru/crs/psfin/PSFinancialsServiceIT.kt